src/index.ts

```diff
@@ -113,7 +114,6 @@ export type {
   SnoozeThreadInput,
   UpdateCustomerCardConfigInput,
   UpsertCustomerInput,
-  UpsertCustomTimelineEntryInput,
```

Comment:
This was removed from the schema since it's deprecated, so had to be removed here as well